CMSgov/onemac
# Oy2 21884 - show kafka topics in UI

Story: https://qmacbis.atlassian.net/browse/OY2-21884
Endpoint: See github-actions bot comment

### Details

Added routes/pages/api for retrieving and viewing kafka topic data

### Changes

- Added routes: event and event/id/date
- Added components: EventList and EventDetail
- Added api endpoints: /getEventList and /getEventDetail

### Test Plan

1. Login as system admin
2. navigate in url to /event
3. verify list defaults to first tab
4. verify list is populated with data
5. verify an item in the list has a linkable id that when clicked goes to the details page
6. verify on details page that the json data appears
